libs/application/templates/parental-leave/src/forms/UploadAdditionalFiles.ts

```diff
@@ -14,39 +17,56 @@ export const UploadAdditionalFiles: Form = buildForm({
   id: 'ParentalLeaveUploadAdditionalFiles',
   title: '',
   logo: Logo,
+  mode: FormModes.IN_PROGRESS,
   children: [
     buildSection({
       id: 'additionalDocuments',
-      title: '',
+      title: parentalLeaveFormMessages.confirmation.title,
       children: [
-        buildFileUploadField({
-          id: 'fileUpload.additionalDocuments',
-          title: parentalLeaveFormMessages.attachmentScreen.genericTitle,
-          introduction:
-            parentalLeaveFormMessages.attachmentScreen.genericDescription,
-          maxSize: FILE_SIZE_LIMIT,
-          maxSizeErrorText:
-            parentalLeaveFormMessages.selfEmployed.attachmentMaxSizeError,
-          uploadAccept: '.pdf',
-          uploadHeader: '',
-          uploadDescription: '',
-          uploadButtonLabel:
-            parentalLeaveFormMessages.selfEmployed.attachmentButton,
+        buildCustomField({
+          id: 'periodSynchronization',
+          title: parentalLeaveFormMessages.confirmation.title,
+          component: 'PeriodSynchronization',
         }),
-        buildSubmitField({
-          id: 'additionalDocuments',
+        buildMultiField({
+          id: 'additionalDocuments1',
```

Comment:
This could be calls: additionalDocumentsScreen

Reply:
Yes I forgot to change that, thanks.